assets/queries/openAPI/2.0/unused_parameter_definition/test/negative1.json

```diff
@@ -0,0 +1,39 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Simple API Overview",
+    "version": "1.0.0"
+  },
+  "paths": {
+    "/": {
+      "get": {
+        "operationId": "listVersionsv2",
+        "summary": "List API versions",
+        "responses": {
+          "200": {
+            "description": "200 response"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/limitParam"
+          }
+        ]
+      }
+    }
+  },
+  "parameters": {
+    "limitParam": {
+      "name": "limit",
+      "in": "body",
+      "description": "max records to return",
+      "required": true,
+      "schema": {
+        "type": "string"
+      },
+      "desc": {
```

Comment:
`desc` does not exist. Maybe we could delete it. The same happens in the other samples

Reply:
@joaoReigota1 take a look at this comment by @rafaela-soares